src/distributions/integer.rs

```diff
@@ -124,8 +124,11 @@ impl_nzint!(NonZeroI128, NonZeroI128::new);
 impl_nzint!(NonZeroIsize, NonZeroIsize::new);
 
 macro_rules! x86_intrinsic_impl {
-    ($($intrinsic:ident),+) => {$(
-        /// Available only on x86/64 platforms
+    ($meta:meta, $doc:literal, $($intrinsic:ident),+) => {$(
+        #[doc = $doc]
+        #[cfg($meta)]
+        // TODO: as doc_cfg/doc_auto_cfg mature ensure they catch this
+        #[cfg_attr(doc_cfg, doc(cfg($meta)))]
```

Comment:
This is redundant:
![image](https://github.com/rust-random/rand/assets/134893/aecd7041-fa24-437e-9c1a-8ca01b4271b9)

Reply:
Are we looking at the same docs? I don't see this on either docs.rs or rust-random.github.io

Reply:
I'm looking at the docs generated locally from this PR.

Reply:
Oh you meant the doc comments are redundant. Yes I'll remove them